src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -518,4 +520,32 @@ def remove_file_post
       end
     end
   end
+
+  describe 'GET #trigger_services' do
+    let(:post_url) { "http://localhost:3200/source/#{source_project}/#{service_package}?cmd=runservice&user=#{user}" }
+    before do
+      login user
+    end
+
+    context 'with right params' do
+      before do
+        get :trigger_services, params: { project: source_project, package: service_package }
+      end
+
+      it { expect(a_request(:post, post_url)).to have_been_made.once }
+      it { expect(flash[:notice]).to eq('Services successfully triggered') }
+      it { is_expected.to redirect_to(action: :show, project: source_project, package: service_package) }
+    end
+
+    context "without a service file in the package" do
+      let(:post_url) { "http://localhost:3200/source/#{source_project}/#{source_package}?cmd=runservice&user=#{user}" }
```

Comment:
Because if you move it more often it's better to move it one layer up. Which in this case would be within "describe 'GET #trigger_services' do".